cEP-0024.md

```diff
@@ -0,0 +1,106 @@
+# Implement GitMate automations and plugins for coala
+
+| Metadata |                                                       |
+| -------- | ----------------------------------------------------- |
+| cEP      | 24                                                    |
+| Version  | 1.0                                                   |
+| Title    | Implement Gitmate automations and plugins for coala   |
+| Authors  | Vamshi Krishna Bommerla <mailto:bommvams72@gmail.com> |
+| Status   | Proposed                                              |
+| Type     | Process                                               |
+
+## Abstract
+
+This cEP describes the improvements to the coala workflow and the necessary
+automation to realise it, as a part of the
+[GSoC project](https://summerofcode.withgoogle.com/projects/#5344316227059712).
+
+## Introduction
+
+[GitMate](https://gitmate.io/) is an automation tool for developers. While it
+works well, there are a number of missing features which would save time of
+our community, for example automatically close PRs from master. Many of them
+are general plugins, which may be useful to other any organisation, but coala
+needs more plugins which support coala development workflow. A team class in
+IGitt, representing GitHub and GitLab teams, will be implemented.
```

Comment:
Please refer https://github.com/coala/cEPs/pull/142/files#r191063287.

Reply:
which may be useful to **other organizations**?